packages/did/test/did.test.js

```diff
@@ -10,12 +11,33 @@ jest.mock(
   { virtual: true }
 );
 
-describe('Idenity registration', () => {
-  it('resolved document MUST be same as published document', async () => {
-    const seed = 'THISISTHESEEDOFTHETICACCOUNTANDISHOULDNOTGIVEITTOANYBODYELSE';
-    const { did, document } = await register({ seed, network: '0x1' });
-    const resolved = await resolver(did);
+let publicKey, did, document, resolved;
+
+beforeAll(async () => {
+  const seed = 'THISISTHESEEDOFTHETICACCOUNTANDISHOULDNOTGIVEITTOANYBODYELSE';
```

Comment:
Is the implementation specific to TIC?

Reply:
The identities are registered with TIC for now.
Is this sample seed make people confused?
Maybe we could give another sample seed, or don't provide the seed to auto-generate the seed?

Reply:
Please follow the manner how Tangle-Accelerator specifies seed. Don't mix with the original seed provided by TIC.

Reply:
Since the Tangle-Accelerator seed is generated randomly, so remove this sample seed and the 
random seed will be generated by the register.